backend/forgather/src/main/java/com/forgather/domain/space/service/PhotoService.java

```diff
@@ -63,6 +67,11 @@ public void saveAll(String spaceCode, List<MultipartFile> multipartFiles) {
 
     private String upload(String spaceCode, MultipartFile multipartFile) {
         try {
+            log.info("{} {} {} {}",
+                logFormatter.formatWithBrackets("Event", "파일 업로드 시작"),
+                logFormatter.formatWithBrackets("SpaceCode", spaceCode),
+                logFormatter.formatWithBrackets("OriginalName", multipartFile.getOriginalFilename()),
+                logFormatter.formatRequestInformation());
```

Comment:
로깅 로직 대부분이 아래와 같은 패턴을 반복하고 있어요.
```
log.info(...
    logFormatter.formatWithBrackets(..);
    ...
    logFormatter.formatRequestInformation());
```
이렇게 되면 로깅을 심을 때마다 개발자는 이런 과정을 추가로 거쳐야 합니다.

1. Slf4j를 클래스에 추가해야겠네
2. 로그레벨을 명시하고... logFormatter에 그 메서드가 뭐더라?
3. 아 `파일 업로드 시작`같은 로그 필드명은 컨벤션이 뭐더라?
4. 음 메서드를 x개 썼으니까 중괄호를 x개 써야겠네

지금 떠오르는 개선방향은 빌더패턴 적용 정도가 있을 것 같습니다. 장점은 다음과 같습니다.
1. formatRequestInformation()과 같은 관용 포맷을 줄일 수 있습니다.
2. 메서드 체인을 통해 가독성을 높일 수 있습니다.

예시
```suggestion
            log.info(
                logFormatter.builder()
                    .event("파일 업로드 시작")
                    .value("SpaceCode", spaceCode)
                    .value("OriginalName", multipartFile.getOriginalFilename())
                    // .requestInformation()는 명시하지 않아도 자동으로 로그에 추가한다
                    .build()
            );

```
추가로 logFormatter를 사용한다면, logger도 따로 만들어보면 어떨까 하는 생각이 듭니다.(지금 급하게 필요한 것은 아닙니다)
현재 구조대로라면 로깅이 필요한 서비스 계층에서는 매번 Slf4j 의존성을 붙여주고 사용해야 합니다. 또한 로그만을 위한 logFormatter 의존성도 추가됩니다.
logger를 분리한다면 로깅의 관심사를 해당 클래스로 떼어낼 수 있고 logFormatter의 접근 범위를 logger 내부로 제한할 수 있을 것 같습니다.

Reply:
동의합니다. 개선해보겠습니다 :)